src/components/unit.tsx

```diff
@@ -1,6 +1,8 @@
 import { Badge } from "react-bootstrap";
 import { unitProps } from "./interface";
 import { HOUSEHOLD_TYPES, STATUS_CODES } from "./util";
+import envsvg from "../assets/env.svg";
```

Comment:
can I suggest that you use meaningful name for both the variable and svg file ? It will be easier for your team to understand what you are trying to import or initialise in one glance.